Makefile

```diff
@@ -20,9 +20,11 @@ VENDOR_MM_SERVER_DIR ?= vendor/github.com/mattermost/mattermost-server/v6
 ENTERPRISE_HASH ?= $(shell cat enterprise_hash)
 TESTFLAGS = -mod=vendor -timeout 30m -race -v
 
+# We specify version for the build; it is the latest semantic version of the tags
+DIST_VER=$(shell git tag -l --sort=-version:refname "v6.3.*" | head -n 1)
 
-PKG=github.com/mattermost/mmctl/commands
-LDFLAGS= -X $(PKG).gitCommit=$(GIT_HASH) -X $(PKG).gitTreeState=$(GIT_TREESTATE) -X $(PKG).buildDate=$(BUILD_DATE)
+PKG=github.com/mattermost/mmctl/v6/commands
```

Comment:
could you elaborate why `v6` is needed?

Reply:
I've cherry picked from https://github.com/mattermost/mmctl/pull/488/files and looks like that line was changed at another commit. Will revert that line

Reply:
I think you are right that you have the `v6` at least in master is there

Reply:
Yes it is on master, but developers did not cherry picked to 6.3 branch. It is applied from 6.4. 

https://github.com/mattermost/mmctl/blob/release-6.4/Makefile

Reply:
I think we need it @isacikgoz right?

Reply:
We moved to versioning on module path therefore we added the `v6` suffix, it's same for the `go.mod` file.

Reply:
`release-6.3` branch does not have go module version hence this needs to be `PKG=github.com/mattermost/mmctl/commands`